openedx/features/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -0,0 +1,368 @@
+                {
+                    "user_attr": "user_group_content_creator",
+                    "expected_status": status.HTTP_403_FORBIDDEN,
+                },
+                {
+                    "user_attr": "user_org_content_creator",
+                    "expected_status": status.HTTP_403_FORBIDDEN,
+                },
+            ),
+        ),
+    )
+    @ddt.unpack
+    def test_list_taxonomy(self, features_dict, test_list):
+        Taxonomy.objects.create(name="Taxonomy enabled 1", enabled=True).save()
+        Taxonomy.objects.create(name="Taxonomy enabled 2", enabled=True).save()
+        Taxonomy.objects.create(name="Taxonomy disabled", enabled=False).save()
+
+        url = reverse("content_tagging:taxonomy-list")
+
+        for test in test_list:
+            user_attr = test["user_attr"]
+            expected_status = test["expected_status"]
+
+            if user_attr:
+                user = getattr(self, user_attr)
+                self.client.force_authenticate(user=user)
+
+            with mock.patch.dict("django.conf.settings.FEATURES", features_dict):
+                response = self.client.get(url)
+            assert response.status_code == expected_status
+
+    # ToDo: Wait definition for taxonomy detail permissions
+    # @ddt.data(
+    #     (None, {"enabled": True}, status.HTTP_403_FORBIDDEN),
+    #     (None, {"enabled": False}, status.HTTP_403_FORBIDDEN),
+    #     (
+    #         "user",
+    #         {"enabled": True},
+    #         status.HTTP_200_OK,
+    #     ),  # ToDo: Verify permission for user
+    #     ("user", {"enabled": False}, status.HTTP_404_NOT_FOUND),
+    #     ("staff", {"enabled": True}, status.HTTP_200_OK),
+    #     ("staff", {"enabled": False}, status.HTTP_200_OK),
+    # )
+    # @ddt.unpack
+    # def test_detail_taxonomy(self, user_attr, taxonomy_data, expected_status):
+    #     create_data = {**{"name": "taxonomy detail test"}, **taxonomy_data}
+    #     taxonomy = Taxonomy.objects.create(**create_data)
+    #     url = reverse("content_tagging:taxonomy-detail", kwargs={"pk": taxonomy.id})
+
+    #     if user_attr:
+    #         user = getattr(self, user_attr)
+    #         self.client.force_authenticate(user=user)
+
+    #     response = self.client.get(url)
+    #     assert response.status_code == expected_status
+
+    #     if status.is_success(expected_status):
+    #         self.assertGreaterEqual(response.data.items(), create_data.items())
+
+    @ddt.data(
+        (
+            {"DISABLE_COURSE_CREATION": False, "ENABLE_CREATOR_GROUP": False},
+            (
+                {"user_attr": None, "expected_status": status.HTTP_403_FORBIDDEN},
+                {"user_attr": "user", "expected_status": status.HTTP_201_CREATED},
+                {
+                    "user_attr": "staff",
+                    "expected_status": status.HTTP_405_METHOD_NOT_ALLOWED, # ToDo: Verify error status 405
```

Comment:
I have some 405 errors (Method not allowed) on create where it should return 201. I will look into it to see what is happening.